cmd/apiKeyValueMapUpload_test.go

```diff
@@ -0,0 +1,98 @@
+package cmd
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRunApiKeyValueMapUpload(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Successfull Api Key Value Map Create Test", func(t *testing.T) {
+		// init
+		config := getDefaultOptions()
+		httpClient := httpMockCpis{CPIFunction: "ApiKeyValueMapUpload", ResponseBody: ``, TestType: "PositiveCase"}
+		// test
+		err := runApiKeyValueMapUpload(&config, nil, &httpClient)
+		// assert
+		if assert.NoError(t, err) {
+			t.Run("check url", func(t *testing.T) {
+				assert.Equal(t, "https://demo/apiportal/api/1.0/Management.svc/KeyMapEntries", httpClient.URL)
+			})
+
+			t.Run("check method", func(t *testing.T) {
+				assert.Equal(t, "POST", httpClient.Method)
+			})
+		}
+	})
+
+	t.Run("Failed case of API Key Value Map Create Test", func(t *testing.T) {
+		// init
+		config := getDefaultOptions()
+
+		httpClient := httpMockCpis{CPIFunction: "ApiKeyValueMapUpload", ResponseBody: ``, TestType: "Negative"}
+
+		// test
+		err := runApiKeyValueMapUpload(&config, nil, &httpClient)
+
+		// assert
+		assert.EqualError(t, err, "HTTP \"POST\" request to \"https://demo/apiportal/api/1.0/Management.svc/KeyMapEntries\" failed with error: 401 Unauthorized")
+	})
+
+	t.Run("Test API Key Value Map payload", func(t *testing.T) {
+		// init
+		config := getDefaultOptions()
+		testPayload := bytes.NewBuffer([]byte(string("{\"encrypted\":true,\"keyMapEntryValues\":[{\"map_name\":\"demoMap\",\"name\":\"demo\",\"value\":\"name\"}],\"name\":\"demoMap\",\"scope\":\"ENV\"}")))
```

Comment:
It's redundant to convert a string to a string (`string("this is already a string)`). Also, the back ticks are a useful way of not needing to escape `"` characters:
```suggestion
		testPayload := bytes.NewBuffer([]byte(`{"encrypted":true,"keyMapEntryValues":[{"map_name":"demoMap","name":"demo","value":"name"}],"name":"demoMap","scope":"ENV"}`))
```

Reply:
fixed

Reply:
fixed